tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRFilePseudolocalizer.java

```diff
@@ -403,7 +403,7 @@ public CLDRFile generate() {
         // Create input CLDRFile object resolving inherited data.
         CLDRFile input = factory.make(ORIGINAL_LOCALE, false);
         XMLSource outputSource = new SimpleXMLSource(outputLocale);
-        for (String xpath : input) {
+        for (String xpath : input.iterableWithoutExtras()) {
```

Comment:
Here, there would be a test failure if it were `iterableDefault`; likewise everywhere that `iterableWithoutExtras` is called outside of CLDRFile